test_common/harness/testHarness.c

```diff
@@ -717,29 +704,41 @@ int callSingleTestFunction( test_definition test, cl_device_id deviceToUse, int
     fflush( stdout );
 
     error = check_opencl_version_with_testname(test.name, deviceToUse);
-    test_missing_feature(error, test.name);
-
-    error = check_functions_for_offline_compiler(test.name, deviceToUse);
-    test_missing_support_offline_cmpiler(error, test.name);
```

Comment:
Are we no longer performing that check?

Reply:
Missed it when cherry-picking from 2.0 branch, now restored.